WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -128,13 +110,20 @@ def update
   end
 
   private def user_params
-    user_params = params.require(:user).permit(*current_user.editable_fields_of_user(user_to_edit))
+    permitted_params = current_user.editable_fields_of_user(user_to_edit).to_a
+    permitted_params.delete :preferred_event_ids
```

Comment:
Hmm... what's this doing? How did `preferred_event_ids` get in there if we don't want it there?